src/overlays/actors/ovl_Magic_Wind/z_magic_wind.c

```diff
@@ -14,8 +14,12 @@ void MagicWind_Init(Actor* thisx, GlobalContext* globalCtx);
 void MagicWind_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void MagicWind_Update(Actor* thisx, GlobalContext* globalCtx);
 void MagicWind_Draw(Actor* thisx, GlobalContext* globalCtx);
+void MagicWind_Shrink(MagicWind* this, GlobalContext* globalCtx);
```

Comment:
very minor but we've been keeping a line break between main functions and action functions in the declarations